examples/fuel-explorer/fuel-explorer/Cargo.toml

```diff
@@ -9,6 +9,7 @@ rust-version = "1.73.0"
 crate-type = ['cdylib']
 
 [dependencies]
+anyhow = "1"
```

Comment:
@lostman 

- So when importing most external crates, we import those through the plugin, so that we don't have to add a ton of dependencies to the indexer itself
- So `anyhow` would be added to `fuel_indexer_plugin/lib.rs` (as well as to that prelude)
- `fuel_indexer_utils` automatically pulls in `fuel_indexer_plugin::prelude` - so if you add `anyhow` to the plugin, it will automatically be available in the indexers 👌🏼 
  - As an example reference, look at how we do this with `bincode`